sos/report/plugins/snapper.py

```diff
@@ -0,0 +1,24 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from glob import glob
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Snapper(Plugin, IndependentPlugin):
+
+    short_desc = 'System snapper'
+
+    plugin_name = 'snapper'
+    profiles = ('system', 'hardware')
+
+    def setup(self):
+
+        self.add_cmd_output('ls -la /usr/lib/snapper/')
```

Comment:
Surely there is more relevant data to collect for `snapper` than just a directory listing?

Taking a quick look there's:

`snapper list-configs`
`snapper get-config` (is this iterable for each config listed from the above command?)
`snapper list` (is there any value is iterating over this list with some other `snapper` command for more info?)

Reply:
ok..right now I will be adding following commands, 
  ls -al /usr/lib/snapper/
 snapper --version
 snapper list
Later will  work on other commands and add the same